lib/graphql/language/lexer.rl

```diff
@@ -37,9 +37,9 @@
   QUOTE =         '"';
   BLOCK_QUOTE =   '"""';
   ESCAPED_BLOCK_QUOTE = '\\"""';
-  BLOCK_STRING_CHAR = (ESCAPED_BLOCK_QUOTE | ^'"""');
+  BLOCK_STRING_CHAR = (ESCAPED_BLOCK_QUOTE | ^QUOTE | ^QUOTE QUOTE{1,2} ^QUOTE);
```

Comment:
Arguably this whole definition should just go in `BLOCK_STRING` now, it's not really a character class or constant sequence.